task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +107,18 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
```

Comment:
You don't need 2 exit conditions here. Leave only condition in the loop statement
![image](https://user-images.githubusercontent.com/7272208/87961005-cc185d80-cabd-11ea-8094-8b962efd89cb.png)